clients/fides-js/src/lib/overlay.module.css

```diff
@@ -45,22 +45,29 @@
   );
 
   /* Everything else */
-  --fides-consent-overlay-font-family: "inherit";
-  --fides-consent-overlay-font-size: 16px;
-  --fides-consent-overlay-font-size-title: 18px;
-  --fides-consent-overlay-padding: 0.75em 2em 1em;
+  --fides-consent-overlay-font-family: Inter, sans-serif;
```

Comment:
Annoying-but-true-fact... we can't rely on `Inter` being available... I'm not 100% sure what to do here yet.

Reply:
yeah, that's why I added `sans-serif` as a [backup option](https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/), which is a browser default. I think it looks reasonable (at least better than Times!). `sans-serif` kicks in in the demo page since that one doesn't have `Inter`

![image](https://github.com/ethyca/fides/assets/24641006/322fe08f-71cd-41ff-b888-455bf6075275)

I was hoping `inherit` would just use whatever font the base website was already using, but that doesn't seem to be the case. could look into it more though!

Reply:
OK, I think that's probably fine to inherit the page default... in theory that works? It's a bit fragile though, so we should consider shipping a font-face with the JS itself. Can you add a follow-up issue for that in the next sprint and we'll just make a plan?

Reply:
It'd also be easy for a customer to override that with a tiny stylesheet like:
```css
:root {
  --fides-consent-overlay-font-family: "Comic Sans"
}
```

...I think

Reply:
https://github.com/ethyca/fides/issues/3440